Testing/Exec/IgnitionDelay/.githubTest.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
@jrood-nrel if this is executed by the CI, should it be somewhere else?

Reply:
I think it's better here. I don't think it should be a hidden file though.